pkg/alert/alert.go

```diff
@@ -1,8 +1,85 @@
 package alert
 
-import "fmt"
+import (
+    "fmt"
+    "errors"
+    "os"
+    "time"
+    "encoding/json"
+    "github.com/anynines/a9s-aha-cli/helpers"
+)
 
-func Stale() error {
-	fmt.Println("FIXME!")
-	return nil
+type Content struct {
+    Idea []struct {
+        Name string `json:"reference_num"`
+        Workflow_status struct {
+            Name string `json:"name"`
+        } `json:"workflow_status"`
+    } `json:"ideas"`
+}
+
+type Req struct {
+    Comments struct {
+        Updated_at string `json:"updated_at"`
+    } `json:"idea"`
+}
+
+func Stale() (error, string) {
+    var status Content
+    var updated_at Req
+    var msg string
+
+    fmt.Println("Query aha.io idea list: ")
+    fmt.Println("Report: ")
+    url := "https://anynines.aha.io/api/v1/ideas?fields=reference_num,workflow_status,created_at"
+    body, err := helpers.HttpGet(url, os.Getenv("AHA_USERNAME"), os.Getenv("AHA_PASSWORD"))
+    if err != nil {
+        errors.New(fmt.Sprintf("Error while executing HTTP Request: %s", err))
+        return nil, ""
+    }
+
+    err = json.Unmarshal([]byte(body), &status)
+    if err != nil {
+        fmt.Println(err)
+        return nil, ""
+    }
+
+    for i, _:= range status.Idea {
+        _, tmp := CheckStale(status, updated_at, i)
+        msg = msg + tmp
+    }
+	return nil, msg
+}
+
+func CheckStale(status Content, updated_at Req, i int) (error, string) {
+    if status.Idea[i].Workflow_status.Name == "Needs review" {
+        url := fmt.Sprintf("https://anynines.aha.io/api/v1/ideas/%s?fields=updated_at",status.Idea[i].Name)
+        body, err := helpers.HttpGet(url, os.Getenv("AHA_USERNAME"), os.Getenv("AHA_PASSWORD"))
+        if err != nil {
+            fmt.Println(err)
+        }
+            
+        err = json.Unmarshal([]byte(body), &updated_at)
+        if err != nil {
+            fmt.Println(err)
```

Comment:
In case of any error, the CLI command should return exit status != 0. You have to deliver errors more or less to cmd/.